src/modules/sensors.cpp

```diff
@@ -325,7 +325,8 @@ static PT_THREAD(sensorsPressureThreadMain(struct pt* pt))
     }
     
     // Alarms
-    if ((control.state != CONTROL_STATE_HOME) && (control.state != CONTROL_STATE_IDLE)) {
+    if ((control.state != CONTROL_STATE_HOME) && (control.state != CONTROL_STATE_IDLE)
+	&& (sensors.calibrated == SENSORS_ALL_CALIBRATED)) {
```

Comment:
This should really only check if the pressure sensor is calibrated. and there should be another one of these for the airflow sensor

```suggestion
	&& (sensors.calibrated & SENSORS_PRESSURE_CALIBRATED)) {
```